index.bs

```diff
@@ -281,7 +281,8 @@ The API is defined by the following Web IDL fragment.
     interface WebAuthentication {
         Promise < ScopedCredentialInfo > makeCredential (
             Account                                 accountInformation,
-            sequence < ScopedCredentialParameters > cryptoParameters,
+            sequence < CredentialType >             supportedTypes,
+            sequence < AlgorithmIdentifier >        supportedAlgorithms,
```

Comment:
My conceptual model here is that there's no relationship between `CredentialType` and `AlgorithmIdentifier`. A relying party would either support an algorithm, or not, and support a credential type, or not. I haven't built an RP. Is this not the case?